net/dns/resolver/tsdns.go

```diff
@@ -62,6 +63,24 @@ type packet struct {
 	addr netaddr.IPPort // src for a request, dst for a response
 }
 
+// ResolverName is the configuration for a DNS resolver.
```

Comment:
sounds like the name of a resolver.

what's wrong with tailcfg.DNSResolver ? seems like this is a copy of that? (why a copy?)